CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix missing `itemtypes` in `connectdTo` methods of injection classes
```

Comment:
It's too technical, it has to be functional to be easily understood by non-developers.